bin/check-test-coverage.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+WATCH_FLAG=false
+
+while getopts ':w' OPTION; do
+	case $OPTION in
+		w)
+		WATCH_FLAG=true
+		shift
+		;;
+	esac
+done
+
```

Comment:
Is this flag needed? I'm trying to think a use case that would involve running watcher when generating a coverage report. It's possible that there are good reasons to leave this that I'm just missing.

Reply:
Oof, nice catch. I don't use this flag at all. Amended in d9dcb5a938c7f2f89f6e3933ae921fec064af07b. Cheers! 👍